### PR TITLE
fizzBuzz Assignment

### PR DESCRIPTION
**Pull request by alexLaws**
I added the fizzBuzz assignment here.  I also made a few cleanup edits to the grid printer file after downloading the pep8 and pyflakes linters.  I think it should be fine now.  I know it's after the fact but if there are any other common errors, would appreciate any feedback!

### CODE REVIEW

**Comment by natasha-aleksandrova** [comment]
@alexLaws  will you please submit your fizzbuzz pr link in EdX

File: students/alexLaws/lesson02/fizzBuzz.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+def fizzBuzz():
```
Comment (natasha-aleksandrova) [inline students/alexLaws/lesson02/fizzBuzz.py]: var name should be `fizz_buzz`

**Comment by alexLaws** [comment]
@natasha-aleksandrova I thought I did submit the pr link in edX

image

**Comment by natasha-aleksandrova** [comment]
I see it now! it wasn't there earlier 🙂

**Comment by robalford** [comment]
Looks good! Thanks for addressing Pep8 violations in grid printer.

**Pull request merged**